.circleci/config.yml

```diff
@@ -100,7 +95,7 @@ build_image: &build_image
     - <<: *exports
     - run: make VERSION="$VERSION" $IMAGE
     - run: |
-        if [[ -z "$CIRCLE_PULL_REQUEST" ]]; then
+        if [[ -n "$DOCKER_USERNAME" && -n "$DOCKER_PASSWORD" ]]; then
```

Comment:
I think we typically check for this in other CI scripts, but you're right that it would be unexpected for them to not be there - and actually that should cause a failure (docker login presumably would exit non-zero).

Reply:
this is for PR. When testing a PR credentials won't be available. The previous condition protected against that and it was more clear, I will restore how it was.

Reply:
Thanks!